### ISSUE TITLE
x/mobile/bind: unsupported named type affects private fields in embedded structure

### ISSUE THREAD

**Issue by ttyh061** [golang/go#14016]
As before we can have a private filed with unsupported named type, but in recently update(gomobile version +24a199a Wed Jan 13), it breaks this rule, that makes a huge impact to code, and make all types which want to be used for mobile narrows down to supported basic type....

Before:

The following type is working

type ABC struct {
    WSClient
}

type WSClient struct {
  conn   *websocket.Conn
}

ABC is the struct exported for use in android.

After:
it is not working with an error message:
gomobile: unsupported named type xxxx

**Comment by hyangah** [comment]
It's because non-pointer struct type as a field is not yet supported.
I don't think it was ever working before but I may be wrong. Duplicate of #11318.

type ABC struct {
   *WSClient
}

will work if you need a workaround :-(
